src/crystal/main.cr

```diff
@@ -4,6 +4,10 @@ lib LibCrystalMain
 end
 
 module Crystal
+  NPROCS = ENV.fetch("NPROCS", "1").to_i
```

Comment:
Nope:
> expected block to return String | ::Nil, not Int32

TBH: this would happen once, at bootstrap, while we're initializing hundreds of other constants and class variables and memory allocations.